chapters/versions.adoc

```diff
@@ -62,13 +73,522 @@ The link:https://docs.vulkan.org/spec/latest/chapters/features.html#features-req
+link:https://registry.khronos.org/vulkan/specs/latest/man/html/VK_KHR_shader_float_controls2.html#_description[Vulkan Spec: Shader Float Controls 2]
+
+* Shader Subgroup Rotate: Rotate values within a subgroup
+* Shader Expect/Assume: Provide hints to the compiler for optimization
+* Shader Float Controls 2: More control over floating-point behavior
+* Vertex Attribute Divisor: Instance-rate vertex attributes
+
+=== Other Notable Features
+
+* Push Descriptor: Update descriptors directly in command buffers (link:https://registry.khronos.org/vulkan/specs/latest/man/html/VK_KHR_push_descriptor.html#_description[Spec])
+* Index Type UINT8: Support for 8-bit indices (link:https://registry.khronos.org/vulkan/specs/latest/man/html/VK_KHR_index_type_uint8.html#_description[Spec])
+* Host Image Copy: Copy image data directly from host memory (link:https://registry.khronos.org/vulkan/specs/latest/man/html/VK_EXT_host_image_copy.html#_description[Spec])
+* Pipeline Protected Access: More flexible protected memory access (link:https://registry.khronos.org/vulkan/specs/latest/man/html/VK_EXT_pipeline_protected_access.html#_description[Spec])
+* Pipeline Robustness: Improved handling of out-of-bounds accesses (link:https://registry.khronos.org/vulkan/specs/latest/man/html/VK_EXT_pipeline_robustness.html#_description[Spec])
+
+== Porting from Vulkan 1.0 to Any Version
+
+This section provides a consolidated guide for porting applications from Vulkan 1.0 to any newer version (1.1, 1.2, 1.3, or 1.4).
+
+[NOTE]
+====
+link:https://docs.vulkan.org/spec/latest/chapters/extensions.html#extendingvulkan-compatibility[Vulkan Spec: Compatibility] |
+link:https://docs.vulkan.org/spec/latest/chapters/extensions.html#extendingvulkan-compatibility-promotion[Vulkan Spec: Extension Promotion]
+====
+
+=== General Porting Checklist
+
+1. *Update SDK and headers*
+   * Ensure you're using the Vulkan SDK that matches your target version or later
+   * Update header files to the latest version
+
+2. *Instance creation*
+   * Update `VkApplicationInfo::apiVersion` to your target version (e.g., `VK_API_VERSION_1_4`)
+   * Use `vkEnumerateInstanceVersion` to check for support of your target version
+
+3. *Device selection*
+   * Check for support of your target version on physical devices
+   * Select devices that support required features for your target version
+
+4. *Feature adoption*
+   * Replace extension usage with core functionality where applicable
+   * Update structure names (e.g., `VkPhysicalDeviceFeatures2KHR` → `VkPhysicalDeviceFeatures2`)
+   * Update function names (e.g., `vkGetPhysicalDeviceFeatures2KHR` → `vkGetPhysicalDeviceFeatures2`)
+   * Enable version-specific features through the appropriate feature structs:
+     - Vulkan 1.1: `VkPhysicalDeviceVulkan11Features`
+     - Vulkan 1.2: `VkPhysicalDeviceVulkan12Features`
+     - Vulkan 1.3: `VkPhysicalDeviceVulkan13Features`
+     - Vulkan 1.4: `VkPhysicalDeviceVulkan14Features`
+
+5. *SPIR-V updates*
```

Comment:
This reads a bit misleading. There is no hard requirement to compile targeting SPIR-V 1.6 with VK 1.4.